### PR TITLE
feature: Cross publish for Scala 2.11

### PR DESCRIPTION
**Pull request by rtfpessoa**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -22,6 +22,12 @@ workflows:
           context: CodacyAWS
           cmd: |
             export VERSION="$(cat .version)"
+            # export SCALA_VERSION="2.12.9"
+            ./gradlew build
```
Comment (andreaTP) [inline .circleci/config.yml]: can't you chain commands?

File: build.gradle [hunk 2]
```diff
@@ -1,6 +1,7 @@
 // Dependencies
-def scalaMajorMinorVersion = "2.12"
-def scalaVersion = (String) "${scalaMajorMinorVersion}.9"
+def scalaVersionParts = System.getenv().getOrDefault('SCALA_VERSION', '2.12.9').tokenize(".")
```
Comment (andreaTP) [inline build.gradle]: so sad ... but if that's the way go for it ...

Comment (rtfpessoa) [reply build.gradle]: if you have any suggestions :disappointed:

File: .circleci/config.yml (hunk 1 above)
Comment (rtfpessoa) [reply .circleci/config.yml]: done

**Pull request merged**